config/grafana/dashboards/integration-service-dashboard.json

```diff
@@ -1310,6 +1311,77 @@
       "yBucketBound": "auto",
       "yBucketNumber": null,
       "yBucketSize": null
+    },
+
```

Comment:
nit: remove newline

Reply:
If this pops up again its due to rebase and i can fix again.